### ISSUE TITLE
Loss function about instance segmentation

### ISSUE THREAD

**Issue by tyaiga** [microsoft/SoftTeacher#72]
How does the loss function of SoftTeacher change when training mask-rcnn, cascade mask-rcnn, and htc++ for instance segmentation?
